dspace_import.py

```diff
@@ -357,31 +370,56 @@ def import_eperson():
             json_p = {'selfRegistered': i['self_registered'], 'requireCertificate': i['require_certificate'],
                       'netid': i['netid'], 'canLogIn': i['can_log_in'], 'lastActive': i['last_active'],
                       'email': i['email'], 'password': i['password']}
+            email2epersonId[i['email']] = i['eperson_id']
             if metadata:
                 json_p['metadata'] = metadata
-            if i['eperson_id'] in user_reg:
-                param = user_reg[i['eperson_id']]
-                param['userRegistration'] = True
-            else:
-                param = {'userRegistration': False}
-            param['selfRegistered'] = i['self_registered']
-            param['lastActive'] = i['last_active']
+            param = {'selfRegistered': i['self_registered'], 'lastActive': i['last_active']}
             try:
                 response = do_api_post('clarin/import/eperson', param, json_p)
                 eperson_id[i['eperson_id']] = convert_response_to_json(response)['id']
+                imported+=1
             except:
                 log('POST request ' + response.url + ' for id: ' + str(i['eperson_id']) +
                     ' failed. Status: ' + str(response.status_code))
 
+        statistics['eperson'] = (len(json_a), imported)
     log("Eperson was successfully imported!")
 
+def import_user_registration():
+    """
+    Import data into database.
+    Mapped tables: user_registration
+    """
+    global eperson_id, email2epersonId, userRegistration_id, statistics
+    imported = 0
+    # read user_registration
+    json_a = read_json("user_registration.json")
+    if json_a:
+        for i in json_a:
+            json_p = {'email': i['email'], 'organization': i['organization'],
+                      'confirmation': i['confirmation']}
+            if i['email'] in email2epersonId:
+                json_p['ePersonID'] = eperson_id[email2epersonId[i['email']]]
+            else:
+                json_p['ePersonID'] = None
+            try:
+                response = do_api_post('clarin/import/userregistration', None, json_p)
+                userRegistration_id[i['eperson_id']] = convert_response_to_json(response)['id']
+                imported+=1
+            except:
+                log('POST request clarin/import/userregistration for id: ' + str(i['eperson_id']) +
+                    ' failed. Status: ' + str(response.status_code))
+
+        statistics['user_registration'] = (len(json_a), imported)
```

Comment:
it is because if the json is empty, it is successful import too, so I wanted to write info into logs about that. 
This is because if the JSON is empty, it is still considered a successful import. Therefore, I wanted to write information into the logs about this and if there would be `not in` and return, we will not write this information into logs.